docs/admin/install.rst

```diff
@@ -63,11 +63,11 @@ install them you can use apt-get:
         python-whoosh python-pil python-libravatar \
         python-babel Git mercurial python-social-auth \
         python-django-compressor python-django-crispy-forms \
-        python-dateutil
+        python-djangorestframework python-dateutil
 
     # Optional for database backend
 
-    apt-get install python-mysqldb   # For MySQL
+    apt-get install python-pymysql   # For MySQL
```

Comment:
Okay, this is Ubuntu specific patch, so we most likely need different instructions for Debian than for Ubuntu, as Django package in Debian doesn't work with python-pymysql...

Quoting Ubuntu package changelog:

>    - debian/patches/pymysql-replacement.patch: Use pymysql as drop in
>      replacement for MySQLdb.
>    - debian/control: Drop python-mysqldb in favor of python-pymysql.